src/ILLink.Shared/TypeSystemProxy/MethodProxy.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ILLink.Shared.TypeSystemProxy
+{
+	internal readonly partial record struct MethodProxy : IMemberProxy
```

Comment:
Is the `record` buying us anything here?

Reply:
Nothing - will remove